crates/matrix-sdk/src/send_queue.rs

```diff
@@ -833,7 +833,7 @@ struct QueueStorage {
     /// All the queued requests that are being sent at the moment.
     ///
     /// It also serves as an internal lock on the storage backend.
-    being_sent: Arc<RwLock<BTreeSet<OwnedTransactionId>>>,
+    being_sent: Arc<RwLock<Option<OwnedTransactionId>>>,
```

Comment:
Maybe the doc comment doesn't match anymore?

Reply:
Good point!